pkg/config/split.go

```diff
@@ -60,7 +60,7 @@ func SplitQuotedFields(in string, quote rune) []string {
 		}
 	}
 
-	if buf.Len() != 0 {
+	if state == inField || buf.Len() != 0 {
```

Comment:
you also need to add a `state = inSpace` after the `buf.Reset()` in `case inField:`

Reply:
Addressed it and added more test cases to cover it.